### ISSUE TITLE
Stores problem

### ISSUE THREAD

**Issue by jaumard** [trailsjs/trailpack-cache#10]
There some weird behavior with stores

If I set this:

```
'use strict'

/**
 * Cache Configuration
 * (app.config.cache)
 *
 */
module.exports = {
  stores: [
    {
      name: 'memory',
      type: 'memory',
      max: 100,
      ttl: 0
    }
  ],
  defaults: ['memory']
}
```

Once server started I have two store my "memory" and a magically appear "memory-store" ^^ I should have only mine, not the default one.

**Comment by jaumard** [comment]
Fixed on https://github.com/trailsjs/trailpack-cache/tree/corrections I'll try to make some other corrections regarding my issues before doing a PR

**Comment by jaumard** [comment]
Fixed on #14
